src/core/datatypes/trees/TopologyNode.cpp

```diff
@@ -1903,6 +2063,48 @@ void TopologyNode::setParent(TopologyNode* p, bool recompute_branch_length)
 }
 
 
+void TopologyNode::setParentAge(double minbl)
```

Comment:
I don't find the name of this function very clear - suggestion `scaleAgesFromTaxonAgesMBL`

Reply:
Good catch; the name is a leftover from an earlier implementation attempt. Changed as suggested.